decompiler/config/all-types.gc

```diff
@@ -34619,7 +34621,7 @@
 ;;(define-extern mc-create-file object) ;; unknown type
 ;;(define-extern get-heap object) ;; unknown type
 ;;(define-extern mc-save object) ;; unknown type
-(define-extern print-game-text function)
+(define-extern print-game-text (function string font-context symbol int int float)) ; TODO decomp error, this seems correct though
```

Comment:
`;; WARN: Failed to convert to atomic ops: Variable could not be constructed from register s7` when trying to decompile this function in `text`

Reply:
Issue here: https://github.com/water111/jak-project/issues/537